keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/InstanceProvider.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.api.ec2
+
+/**
+ * Extensible mechanism allowing for abstraction of details of the VM instance (e.g.
+ * [LaunchConfigurationSpec.instanceType], [LaunchConfigurationSpec.ebsOptimized], etc.)
+ */
+interface InstanceProvider
```

Comment:
I couldn't find where this is being used. Can you give me a pointer, please?

Reply:
Oh yeah. That's the new interface for the instance type recommender plugin that I needed to make all these changes to enable.